assets/src/blocks/Media/MediaEditor.js

```diff
@@ -0,0 +1,155 @@
+import { Fragment, useEffect, useState, useCallback } from "@wordpress/element";
+import { Tooltip, PanelBody } from '@wordpress/components';
+import { MediaPlaceholder } from "@wordpress/editor";
+import { InspectorControls } from '@wordpress/block-editor';
+
+import { URLInput } from "../../components/URLInput/URLInput";
+import { MediaEmbedPreview } from "./MediaEmbedPreview";
+import { MediaElementVideo } from './MediaElementVideo';
+import { useSelect } from '@wordpress/data';
+import { debounce } from 'lodash';
+
+const { __ } = wp.i18n;
+const { RichText } = wp.blockEditor;
+
+const renderEdit = ({ attributes, setAttributes }) => {
+  // Using a state to prevent the input losing the cursor position, a React issue reported multiple times
+  const [ mediaURL, setMediaURL ] = useState(attributes.youtube_id);
+  const debouncedMediaURLUpdate = useCallback(debounce(value => setAttributes({ youtube_id: value }), 300), []);
+
+  return (
+    <Fragment>
+      <InspectorControls>
+        <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+          <URLInput
+            label={__('Media URL/ID', 'planet4-blocks-backend')}
+            placeholder={__('Enter URL', 'planet4-blocks-backend')}
+            value={ mediaURL }
+            onChange={
+              value => {
+                setMediaURL(value)
+                debouncedMediaURLUpdate(value)
+              }
+            }
+            help={__('Can be a YouTube, Vimeo or Soundcloud URL or an mp4, mp3 or wav file URL.', 'planet4-blocks-backend')}
+          />
+        </PanelBody>
+      </InspectorControls>
+    </Fragment>
+  );
+}
+
+const renderView = (props, toAttribute) => {
+  const { attributes, setAttributes } = props;
+  const { video_title, description, embedHtml, youtube_id, posterURL } = attributes;
+
+  function onSelectImage({id}) {
+    setAttributes({video_poster_img: id});
+  }
+
+  function onSelectURL({url}) {
+    setAttributes({id: null});
```

Comment:
Is this `null` a mistake? Shouldn't it be `url`?

Reply:
Yep, I just ported this here and forgot about it. I just checked, and this is also present in the old Media Block, selecting a URL using the "Insert via URL" button does nothing (!). I removed this listener and the `onSelectURL` property from the component, which prevents the button from rendering at all. 

cc @comzeradd